docs/resources/cbh_instance.md

```diff
@@ -69,203 +68,54 @@ The following arguments are supported:
   Changing this parameter will create a new resource.
 
 * `period_unit` - (Required, String, ForceNew) Specifies the charging period unit of the instance.
-  Valid values are *month* and *year*. This parameter is mandatory if `charging_mode` is set to *prePaid*.
+  Valid values are *month* and *year*.
 
   Changing this parameter will create a new resource.
 
 * `period` - (Required, Int, ForceNew) Specifies the charging period of the read replica instance.
   If `period_unit` is set to **month**, the value ranges from 1 to 9.
   If `period_unit` is set to **year**, the value ranges from 1 to 3.
-  This parameter is mandatory if `charging_mode` is set to **prePaid**.
-
-  Changing this parameter will create a new resource.
-
-* `auto_renew` - (Required, String, ForceNew) Specifies whether auto renew is enabled.
-  Valid values are "true" and "false". Defaults to **false**.
-
-  Changing this parameter will create a new resource.
-
-* `image_id` - (Optional, String, ForceNew) Specifies a image ID.
-
-  Changing this parameter will create a new resource.
-
-* `user_data` - (Optional, String, ForceNew) Specifies the inject user data.
-
-  Changing this parameter will create a new resource.
-
-* `password` - (Optional, String) Specifies the initial password.
-
-* `key_name` - (Optional, String, ForceNew) Specifies the secret key of the admin.
-
-  Changing this parameter will create a new resource.
-
-* `vpc_id` - (Optional, String, ForceNew) Specifies the ID of a VPC.
 
   Changing this parameter will create a new resource.
 
-* `ip_address` - (Optional, String) Specifies the IP address of the subnet.
+* `auto_renew` - (Optional, String) Specifies whether auto renew is enabled.
+  Valid values are **true** and **false**. Defaults to **false**.
 
-* `public_ip` - (Optional, List)
-  The [PublicIP](#CBHInstance_PublicIP) structure is documented below.
+* `subnet_address` - (Optional, String) Specifies the IP address of the subnet.
 
-* `root_volume` - (Optional, List, ForceNew)
+* `password` - (Required, String) Specifies the password for logging in to the management console.
```

Comment:
make the required field before the Optional

Reply:
done